docs/walkthrough/use-promitor-with-managed-identity.md

```diff
@@ -0,0 +1,487 @@
+spec:
+  azureIdentity: ${AD_POD_IDENTITY_NAME}
+  selector: ${AD_POD_IDENTITY_NAME}
+EOF
+```
+
+You can verify that the resources were created successfully as following:
+
+```bash
+$ kubectl get azureidentity
+$ kubectl get azureidentitybinding
+```
+
+### Verifying the AAD Pod Identity installation
+
+Before going further, we will check if our AAD Pod Identity is deployed & configured correctly.
+
+We will spin up a pod and use the Azure CLI to authenticate:
+
+```bash
+kubectl run azure-cli -it --image=mcr.microsoft.com/azure-cli --labels=aadpodidbinding=$AD_POD_IDENTITY_NAME /bin/bash
+```
+
+Note that we are adding `aadpodidbinding` as a label, which is linking the pod to the AAD Pod Identity binding
+ that we have just created.
+
+> _**Warning:**_ It can take some times to Aad Pod Identity to bind the identity to your deployed container.
+> If you encountered an error, relaunch the `az login -i --debug` command after 60 sec.
+
+Next, we will run `az login -i --debug` to see the different steps it takes:
+
+```bash
+# Once you are log in the container, and have the bash command line available, try to login using the Managed Identity:
+# If you don't see a command prompt, try pressing enter.
+$ bash-5.0# az login -i --debug
+msrestazure.azure_active_directory: MSI: Retrieving a token from http://169.254.169.254/metadata/identity/oauth2/token, with payload {'resource': 'https://management.core.windows.net/', 'api-version': '2018-02-01'}
+msrestazure.azure_active_directory: MSI: Token retrieved
+cli.azure.cli.core._profile: MSI: token was retrieved. Now trying to initialize local accounts...
+...
+[
+  {
+    "environmentName": "AzureCloud",
+    "homeTenantId": "c8819874-9e56-4e3f-b1a8-1c0325138f27",
+    "id": "0f9d7fea-99e8-4768-8672-06a28514f77e",
+    "isDefault": true,
+    "managedByTenants": [
+      {
+        "tenantId": "2f4a9838-26b7-47ee-be60-ccc1fdec5953"
+      }
+    ],
+    "name": "Visual Studio Enterprise",
+    "state": "Enabled",
+    "tenantId": "c8819874-9e56-4e3f-b1a8-1c0325138f27",
+    "user": {
+      "assignedIdentityInfo": "MSI",
+      "name": "systemAssignedIdentity",
+      "type": "servicePrincipal"
+    }
+  }
+]
+```
+
+If your Azure CLI container is able to retrieve some information from Azure
+without having to log in with your credentials,
+it means the CLI is using your Managed Identity, through the Pod Identity Binding.
+
+You may have a result indicating you are log in, using a **System Assigned Identity**
+
+```json
+"user": {
+    "assignedIdentityInfo": "MSI",
+    "name": "systemAssignedIdentity",
+}
+```
+
+## Deploying Promitor with Managed Identity
+
+### Create a metrics declaration for Promitor
+
+Before deploying Promitor, we will create a values file for our Helm deployment.
+
+In the configuration, we define what Azure resource that we want to scrape and that we want to use managed identity.
+
+Here is an example:
+
+```yaml
+azureAuthentication:
+  mode: UserAssignedManagedIdentity
```

Comment:
@Mimetis When should people use user assignedi dentity with the id here?

Reply:
When they are using a managed identity, not backed by Aad Pod Identity

Reply:
That's what I've figured, thank you for clarifying and contributing!!